src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,108 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double LIST_EXPAND = 0.5;
+    private Object[] elementData;
+    private int size;
+
+    public ArrayList(int number) {
```

Comment:
"number" name doesn't explain what is this variable for.
```suggestion
    public ArrayList(int initialCapacity) {
```